TypeCobol.Test/Parser/Programs/Cobol85/ReferenceModifiers.rdz.cbl

```diff
@@ -0,0 +1,64 @@
+﻿       IDENTIFICATION DIVISION.
+       PROGRAM-ID. Pgm.
+       DATA DIVISION.
+       WORKING-STORAGE SECTION.
+       01 item   PIC X(20).
+       01 part   PIC X(20).
+       01 var0   PIC 9 VALUE 0.
+       01 var1   PIC 9 VALUE 1.
+      
+       PROCEDURE DIVISION.
+      * OK this is a shorthand
+           MOVE item(2:) TO part
+      * KO zero position
+           MOVE item(0:) TO part
+      * KO negative position
+           MOVE item(-1:) TO part
+      * KO by a previous step
+      *     MOVE item(:2) TO part
```

Comment:
You mean it's a scanner error ?

Reply:
This error must have been caught by the cup parser at this point.
The line has been uncommented to let the diagnostic appear.